core/pom.xml

```diff
@@ -37,6 +37,12 @@
         <dependency>
             <groupId>io.confluent</groupId>
             <artifactId>kafka-connect-avro-data</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.glassfish.jersey.core</groupId>
+                    <artifactId>jersey-common</artifactId>
```

Comment:
Seems like something that would be solved in the avro-data package. Also, is the removal safe?

Reply:
All the tests passed for the repo. Is there any other way to check if it's a safe change?

Reply:
Not convinced we should pin this to a higher/different version. 
The most recent fix that sets the version in avro data is: https://github.com/confluentinc/rest-utils/pull/246
If that's what the rest of CP is using maybe it makes sense to stay in sync.